packages/cli/src/commands/sandbox/metrics.ts

```diff
@@ -66,7 +66,7 @@ export const metricsCommand = new commander.Command('metrics')
           console.log(`\nMetrics for sandbox ${asBold(sandboxID)}:`)
         }
 
-        const isRunningPromise = listSandboxes()
+        const isRunningPromise = listSandboxes({ state: ['running'] })
```

Comment:
I think we can check this more efficiently now, you can do sandbox get, right?

Reply:
`Sandbox.getInfo` is not in the beta branch